spec/ddtrace/contrib/rails/rack_spec.rb

```diff
@@ -112,6 +139,19 @@ def internal_server_error
   let(:request_span) { spans[0] }
   let(:controller_span) { spans[1] }
```

Comment:
Minor: I noticed this causes a bit of inconsistency in the specs; there are a few where we go `request_span, controller_span, ... = spans` and others where we use this let. I would suggest using the same style on every spec :)

Reply:
I'll remove these, and I think that most tests will want to manually spans their spans.